src/ls.js

```diff
@@ -1,5 +1,6 @@
 var path = require('path');
 var fs = require('fs');
+var glob = require('glob');
```

Comment:
@nfischer: good point. When I have a spare moment (I'm busy for the rest of today, sorry!), I'll do that.